package.json

```diff
@@ -55,9 +55,11 @@
     "link-list": "npm ls --depth=0 --link=true"
   },
   "dependencies": {
-    "@babel/runtime": "^7.20.13",
-    "@kitware/vtk.js": "26.5.6",
-    "core-js": "^3.2.1"
+    "@babel/runtime": "7.16.3",
+    "config-point": "^0.3.4",
+    "core-js": "^3.2.1",
+    "cornerstone-core": "2.6.0",
+    "wslink": "^0.1.8"
```

Comment:
I'm not sure which of these are necessary? they seem wrong to me. We don't have dependency on core-js, and you are down grading babel and adding config point ....

Reply:
Apart from these 2, do you notice anything else?

Reply:
we shouldn't add any dependencies that was not there before? do you need these to be added for this PR?

Reply:
Okay. I am not sure if these are added by Lerna 6.x (I hope not).  But I haven't got a chance to checkout this yet. Will let you know once I do

Reply:
Sorry, didn't get a chance to work on this. Will update the PR today

Reply:
@sedghi Check now. Once the CI passes, you can merge this